apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -61,6 +65,15 @@ const placeholderSession = {
   endTime: '5:00pm',
 };
 
+const ALL_PL_TOPICS = [
+  'Test Self Paced PL Topic',
+  'Another Test Self Paced PL Topic',
+  'A Third Test Self Paced PL Topic',
+  'And a Final Very Long Test Self Paced PL Topic That Takes Up a Line',
+];
+
+const INPUT_HEIGHT = 34;
```

Comment:
Nit: since this is only used in one spot in the in-file styling, I imagine it's fine to just set that height style down below to 34